File: application/models/Nomenclature.php
<?php

class Application_Model_Nomenclature extends Application_Model_Element_Abstract {
    protected $_idField = 'nomid';
    protected $_nomlib;
    protected $_nomdate;
    protected $_nomtemps;
    protected $_nomnbchaine;
    protected $_nomqr;
    protected $_nomprix;
    protected $_nomdes;
    protected $_nomsuppr;
    
    public function getNomlib() {
        return $this->_nomlib;
    }

    public function setNomlib($nomlib) {
        $this->_nomlib = $nomlib;
    }

    public function getNomdate() {
        return $this->_nomdate;
    }

    public function setNomdate($nomdate) {
        $this->_nomdate = $nomdate;
    }

    public function getNomtemps() {
        return $this->_nomtemps;
    }

    public function setNomtemps($nomtemps) {
        $this->_nomtemps = $nomtemps;
    }

    public function getNomnbchaine() {
        return $this->_nomnbchaine;
    }

    public function setNomnbchaine($nomnbchaine) {
        $this->_nomnbchaine = $nomnbchaine;
    }

    public function getNomqr() {
        return $this->_nomqr;
    }

    public function setNomqr($nomqr) {
        $this->_nomqr = $nomqr;
    }

    public function getNomprix() {
        return $this->_nomprix;
    }

    public function setNomprix($nomprix) {
        $this->_nomprix = $nomprix;
    }

    public function getNomdes() {
        return $this->_nomdes;
    }

    public function setNomdes($nomdes) {
        $this->_nomdes = $nomdes;
    }

    public function getNomsuppr() {
        return $this->_nomsuppr;
    }

    public function setNomsuppr($nomsuppr) {
        $this->_nomsuppr = $nomsuppr;
    }

    
}

?>

File: src/models/Nomenclature.java
package models;


import java.math.BigDecimal;
import java.util.Date;
import javax.persistence.*;
@Entity
@Table(name="nomenclature")
public class Nomenclature {
	
    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private int nomid;
    
    @Column(name="nomlib", columnDefinition="VARCHAR(250)", nullable=true)
    private String nomlib ;
    
    @Column(name="nomdate", columnDefinition="DATE", nullable=true)
    @Temporal(javax.persistence.TemporalType.DATE)
    private Date nomdate ;

    @Column(name="nomtemps", columnDefinition="INTEGER", nullable=true)
    private int nomtemps ;

    @Column(name="nomnbchaine", columnDefinition="INTEGER", nullable=true)
    private int nomnbchaine ;
    
    @Column(name="nomqr", columnDefinition="VARCHAR(250)", nullable=true)
    private String nomqr ;
    
    @Column(name="nomprix", columnDefinition="NUMERIC(20,2)", nullable=true)
    private BigDecimal nomprix ;

    @Column(name="nomdes", columnDefinition="VARCHAR(250)", nullable=true)
    private String nomdes ;
    
      
    @Column(name="nomsuppr", columnDefinition="BOOLEAN", nullable=true)
    private boolean nomsuppr ;

    public int getNomid() {
        return nomid;
    }

    public void setNomid(int nomid) {
        this.nomid = nomid;
    }

    public String getNomlib() {
        return nomlib;
    }

    public void setNomlib(String nomlib) {
        this.nomlib = nomlib;
    }

    public Date getNomdate() {
        return nomdate;
    }

    public void setNomdate(Date nomdate) {
        this.nomdate = nomdate;
    }

    public int getNomtemps() {
        return nomtemps;
    }

    public void setNomtemps(int nomtemps) {
        this.nomtemps = nomtemps;
    }

    public int getNomnbchaine() {
        return nomnbchaine;
    }

    public void setNomnbchaine(int nombnchaine) {
        this.nomnbchaine = nombnchaine;
    }

    public String getNomqr() {
        return nomqr;
    }

    public void setNomqr(String nomqr) {
        this.nomqr = nomqr;
    }

    public BigDecimal getNomprix() {
        return nomprix;
    }

    public void setNomprix(BigDecimal nomprix) {
        this.nomprix = nomprix;
    }

    public String getNomdes() {
        return nomdes;
    }

    public void setNomdes(String nomdes) {
        this.nomdes = nomdes;
    }

    public boolean isNomsuppr() {
        return nomsuppr;
    }

    public void setNomsuppr(boolean nomsuppr) {
        this.nomsuppr = nomsuppr;
    }
    
    
    
}